client/src/pages/WhatIsAutism/WhatIsAutism.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import ReactPlayer from "react-player/youtube";
+import { BsPlayCircleFill } from "react-icons/bs";
+import whatIsAutismImg from "../../assets/images/what-is-autism.jpeg";
+import whatIsAutismVideoImg from "../../assets/images/what-is-autism-video.jpg";
+import FooterNav from "../Home/FooterNav/FooterNav";
+import FooterIcon from "../Home/FooterIcon/FooterIcon";
+import FooterLogo from "../Home/FooterLogo/FooterLogo";
+import "./WhatIsAutism.css";
+
+function WhatIsAutism() {
+	return (
+		<div className="what-is-autism-container">
+			<img
+				className="hero-image"
+				src={whatIsAutismImg}
+				alt="A woman and a girl are talking"
+			></img>
+			<div className="main-container">
+				<div className="discription-container">
+					<div className="discription">
+						Autism or Autism Spectrum Disorder is not an illness or a disease.
```

Comment:
i'd suggest making it a bunch of <p> elements instead of separating them with line-break <br/>